sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -920,6 +920,11 @@ public void setBypassSecurity(boolean bypassSecurity) {
 
   /** SentryOptions ctor It adds and set default things */
   public SentryOptions() {
+    // UncaughtExceptionHandlerIntegration should be inited before everything.
+    // if there's an error on the setup, we are able to capture it
+    integrations.add(new UncaughtExceptionHandlerIntegration());
```

Comment:
worth noting this only adds the instance. `register` will be called later on so adding the instance to the list vs initializing the ExecutionService doesn't really make a difference.

L: Maybe better adding the executtorService initialization above this so it's not confusing for the reader?

Reply:
yeah comparing to the ExecutionService, true.
I meant to be before every other integration.

Reply:
true